src/main/java/Task.java

```diff
@@ -0,0 +1,148 @@
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.PrintStream;
+import java.io.Serializable;
+import java.io.IOException;
+import java.io.File;
+import java.util.ArrayList;
+import java.io.ObjectInputStream;
+import java.io.FileInputStream;
+import java.util.Scanner;
+
+
+public class Task implements Serializable {
+    protected String description;
+    protected boolean isDone;
+    protected static int taskCounter = 0;
```

Comment:
Can consider just creating a method to return taskArrList.size() instead of manually increasing the counter and decreasing it each time a task is added or removed.